cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -205,6 +205,8 @@ module Cardano.Api.Typed (
     deserialiseFromTextEnvelope,
     readFileTextEnvelope,
     writeFileTextEnvelope,
+    readTextEnvelopeFromFile,
+    readTextEnvelopeOfTypeFromFile,
```

Comment:
I don't really like these names, but I plan to revise and refactor the `TextView` and TextEnvelope` stuff in a later PR.